src/pipecat/services/openai.py

```diff
@@ -212,7 +211,7 @@ async def process_frame(self, frame: Frame, direction: FrameDirection):
             context = OpenAILLMContext.from_image_frame(frame)
         elif isinstance(frame, LLMModelUpdateFrame):
```

Comment:
theoretically, this can now move into `LLMService` and be another thing these processors don't have to remember to do

Reply:
Makes sense.